packages/core/src/components/TabsSlider/Tab/types.ts

```diff
@@ -0,0 +1,29 @@
+import { TabsSliderTheme } from '@medly-components/theme';
+import { HTMLProps, WithThemeProp } from '@medly-components/utils';
+
+export type StyledProps = HTMLProps<HTMLButtonElement> &
+    WithThemeProp &
+    TabsSliderTheme & {
+        active?: boolean;
+        fraction?: any;
+        totalTabs?: number;
+    };
+
+export interface Props extends HTMLProps<HTMLButtonElement> {
+    /** Ids of all tabs */
```

Comment:
I guess it should be total number of tabs

Reply:
Good catch - updated in NoApp PR (https://github.com/medlypharmacy/noapp-app/pull/1)